src/components/home/PanelResult.vue

```diff
@@ -53,6 +130,17 @@
               </v-toolbar-title>
             </v-toolbar>
           </template>
+          <template v-slot:[`item.institution`]="{ item }">
+            <v-tooltip bottom>
+              <template v-slot:activator="{ on }">
```

Comment:
[vue/no-unused-vars]
'on' is defined but never used.
You're missing the v-on="on" in the element doing the activation of the tooltip

Reply:
updated